src/components/Tooltip/Tooltip.js

```diff
@@ -7,6 +7,7 @@
 
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import * as ReactIs from 'react-is';
```

Comment:
Can we bring in only the specific import we need?

Reply:
yep, I have just committed it.